runtime/doc/vim_diff.txt

```diff
@@ -383,6 +383,8 @@ Normal commands:
 Options:
   'ttimeout', 'ttimeoutlen' behavior was simplified
   |jumpoptions| "stack" behavior
+  |jumpoptions| "view" behavior which makes navigation less confusing, and
+  also applies to |mark-motions|.
```

Comment:
IMO "less confusing" is too vague and doesn't actually describe what `"view"` does.

Reply:
Agree about dropping `Which`, should also read `tries to restore` (missing `to`).